Multigeo site enumeration API support (#19923)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/site-list.md

|`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.

In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.

> If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].

Reviewer: ```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites][].
```

[$search]: site-search.md
[sites]: ../resources/site.md

> [site-getallsites]: ../api/site-getallsites.md

Reviewer: ```suggestion
[getallsites]: ../api/site-getallsites.md
```

|`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.

In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.

> If you want to list all sites across all geographies, refer to [site-getallsites.md][].

Reviewer: ```suggestion
If you want to list all sites across all geographies, refer to [getallsites][../api/site-getallsites.md].
```

> If you want to list all sites across all geographies, refer to [sites-getallsites.md][].

Reviewer: ```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites.md][].
```

> If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].

Reviewer: ```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites][].
```

[$search]: site-search.md
[sites]: ../resources/site.md

> [site-getallsites.md]: site-getallsites.md

Reviewer: ```suggestion
[getallsites]: ../api/site-getallsites.md
```

> [sites-getallsites.md]: sites-getallsites.md

Reviewer: ```suggestion
[site-getallsites.md]: site-getallsites.md
```

> [getallsites]: ../api/site-getallsites.md

Reviewer: ```suggestion
[site-getallsites]: ../api/site-getallsites.md
```

> For more guidance about building applications that use site discovery for scanning purposes, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online&preserve-view=true).

Reviewer: ```suggestion
For more guidance about building applications that use site discovery for scanning purposes, see [Best practices for discovering files and detecting changes at scale](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance).
```
